modules/datadict/README.md

```diff
@@ -0,0 +1,43 @@
+# Data Dictionary
+
+## Purpose
+
+The data dictionary module is used for viewing and modifying the
+LORIS data dictionary.
+
+## Scope
+
+The module displays and manages the data dictionary for things
+already stored in the LORIS `parameter_type` table. This table is
+generally autopopulated and provides a dictionary for imaging headers
+(populated by the imaging pipeline) and behavioural instrument
+(populated by the `lorisform_parser.php` script).
+
+It does not provide a way to enter new things into the data dictionary,
+or describe the data dictionary for arbitrary SQL tables which are
+not stored in the `parameter_type` table, such as instrument flags
+or candidate/session data.
+
+## Permission
+
+The `data_dict_view` permission allows the user to view the LORIS
+data dictionary.
+
+The `data_dict_edit` permission allows the user to both view, and
+modify the description of, rows in the data dictionary.
+
+## Configuration
+
+None
+
+## Interactions with LORIS
+
+The content for the data dictionary module comes from the
+`parameter_type` table, which most often is populated for instruments
+by the `lorisform_parser.php` script.
```

Comment:
I'd keep the `parameter_type` table note here, and : 
* move the population (setup) step up to Configuration section where it belongs 
* Note that new categories of parameter type can be added in the `parameter_type_category` table.